sourced/ml/cmd/args.py

```diff
@@ -91,3 +91,9 @@ def add_bow_args(my_parser: argparse.ArgumentParser):
     my_parser.add_argument(
         "--batch", default=BOWWriter.DEFAULT_CHUNK_SIZE, type=int,
         help="The maximum size of a single BOW file in bytes.")
+
+
+def add_dzhigurda_arg(my_parser):
+    my_parser.add_argument(
+        "--dzhigurda", default=0, type=int,
+        help="Index of the examined commit in the history.")
```

Comment:
It used to be the number of commits to extract starting from the top level. Did we change it?

Reply:
No, and it is just copy-pasted help. Will fix it.